src/dispatchers/touchDispatcher.ts

```diff
@@ -2,7 +2,7 @@
 
 module Plottable {
 export module Dispatcher {
-  export type TouchCallback = (p: Point, e: TouchEvent) => any;
+  export type TouchCallback = (points: Point[], ids: number[], e: TouchEvent) => any;
```

Comment:
From the TouchEvent perspective, let's say that you put two fingers down on a mobile device and color a point under one finger red and the other blue.  Now let's move these fingers around.  How does the application know that the red circle is correlated with one finger and the blue is with the other?  It is because a Touch object's ID essentially notes the instance of the Touch